content/reference/drops/search_form_drop.md

```diff
@@ -0,0 +1,36 @@
+# Search Form Drop
+
+Return attributes for search form.
+
+#### Example
+
+~~~ liquid
+  <select class="select optional form-control chosen-select" data-placeholder="{% t 'search.form.destination' %}" id="search_destination" multiple="multiple" name="destination[]" placeholder="{% t 'search.form.destination' %}">
+    {% for destination in search_form.destinations %}
+      <option
+        {% if destination.selected %}
+          selected="selected"
+        {% endif %}
+        value="{{ destination.id }}">
+        {{ destination.name }} ({{ destination.count }})
+      </option>
+    {% endfor %}
+  </select>
+~~~
+
+Available attributes:
+
+* amenities
+* basic_amenities
+* checkin
+* checkout
+* destinations
+* guests
+* guests_choices
+* min_price_for_stay
+* max_price_for_stay
+* length_of_stay
+* rentals_tags
+* rental_types
+* search_url
+* selected_destinations
```

Comment:
I think couple of more here from the delegations, `dislay_type`, `min_price`, etc.